zbus_macros/src/iface.rs

```diff
@@ -74,11 +74,11 @@ def_attrs! {
 }
 
 #[derive(Debug, Default)]
-struct Property<'a> {
+struct Property {
     read: bool,
     write: bool,
     emits_changed_signal: PropertyEmitsChangedSignal,
-    ty: Option<&'a Type>,
+    ty: Option<Type>,
```

Comment:
Why is this change needed?

Reply:
The setter's type is inferred from a value from the `method_info` that was dropped in the loop. So I need to copy it rather than reference it.